OpenRA.Mods.Common/Traits/BotModules/Squads/States/ProtectionStates.cs

```diff
@@ -9,18 +9,41 @@
  */
 #endregion
 
+using System.Linq;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits.BotModules.Squads
 {
-	class UnitsForProtectionIdleState : GroundStateBase, IState
+	abstract class ProtectionStateBase : GroundStateBase
+	{
+	}
```

Comment:
Weird placement of the brackets.

Reply:
Oh, that one is to help it conflict less when merged before #17999. (I hope this PR merged first so conflict should be little here)

Reply:
On a second thought I think the conflict will always be here so I change that to code format.

Reply:
fixed